mock-server-cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file doesn't belong here. Please move to `tests/utils/mock-server/`

Reply:
done, thanks